CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.10)
+cmake_minimum_required(VERSION 3.14)
```

Comment:
a jaka wersja cmake jest na wirtualce ?
Żeby sie nie okazało, że nagle wszystkim przestanie to działać.
Warto dodac informacje do readme, ze wersja cmake musi by 3.14

Ale skonsultuj to z innymi grupami wczesniej, zeby wiedzieli o tej zmianie i o tym, ze musza updatowac ewentualnie swojego cmake

Reply:
> a jaka wersja cmake jest na wirtualce ?

Nie wiem jaka jest na świeżej, ale na zaktualizowanej powinna być co najmniej 3.14 (w końcu się buduje). To nie jest tak, że ta pojedyncza zmiana powoduje zależność od wersji 3.14, ona powstała już wcześniej. Ta zmiana sprawia, że cmake od razu wypluje, że ma za niską wersje, a nie udławi się nie mogąc znaleźć modułu FindSQLite3. Nikt nie zgłaszał żadnych błędów z tym związanym na kanonicznej wirtualce. Jutro mogę jeszcze sprawdzić osobiście.

Nie jest to najlepsze rozwiązanie problemu (w zasadzie to w ogóle nim nie jest). Na przykład cmake dostępny w repo Debiana Stable niestety tego projektu nie zbuduje. Spróbuję usunąć zależność od wersji 3.14. Jednak jako tymczasowe "rozwiązanie" po prostu wrzuciłbym to + uwagę w README odnośnie wymaganej wersji cmake.

Reply:
spoko, to dodaj do readme notke